patches/server/0022-Monumenta-Mob-behavior-changes.patch

```diff
@@ -0,0 +1,197 @@
++
++        this.targetSelector.addGoal(2, new net.minecraft.world.entity.ai.goal.target.HurtByTargetGoal(this, new Class[0]));
++        this.targetSelector.addGoal(2, new net.minecraft.world.entity.ai.goal.target.NearestAttackableTargetGoal<>(this, Player.class, true));
++    }
++
++    @Override
++    protected SoundEvent getAmbientSound() {
++        return SoundEvents.ZOMBIE_AMBIENT;
++    }
++
++    @Override
++    protected SoundEvent getHurtSound(DamageSource source) {
++        return SoundEvents.ZOMBIE_HURT;
++    }
++
++    @Override
++    public SoundEvent getDeathSound() {
++        return SoundEvents.ZOMBIE_DEATH;
++    }
++
++    protected SoundEvent getStepSound() {
++        return SoundEvents.ZOMBIE_STEP;
++    }
++
++    @Override
++    protected void playStepSound(BlockPos pos, BlockState state) {
++        this.playSound(this.getStepSound(), 0.15F, 0.3F);
++    }
++
++    @Override
++    public MobType getMobType() {
++        return MobType.UNDEAD;
++    }
+ }
+diff --git a/src/main/java/net/minecraft/world/entity/monster/Zombie.java b/src/main/java/net/minecraft/world/entity/monster/Zombie.java
+index 35f0203d260c11b729c30e6241316fda4b70bfd7..cf74f1d90113e2301e83f2dbc507ce7ba0a55dd7 100644
+--- a/src/main/java/net/minecraft/world/entity/monster/Zombie.java
++++ b/src/main/java/net/minecraft/world/entity/monster/Zombie.java
+@@ -225,10 +225,11 @@ public class Zombie extends Monster {
+             } else if (this.convertsInWater()) {
+                 if (this.isEyeInFluid(FluidTags.WATER)) {
+                     ++this.inWaterTime;
+-                    if (this.inWaterTime >= 600) {
+-                        this.startUnderWaterConversion(300);
+-                        this.lastTick = MinecraftServer.currentTick; // Paper - Make sure this is set at start of process - GH-1887
+-                    }
++                    // Monumenta - disable drowned conversion
++//                    if (this.inWaterTime >= 600) {
++//                        this.startUnderWaterConversion(300);
++//                        this.lastTick = MinecraftServer.currentTick; // Paper - Make sure this is set at start of process - GH-1887
++//                    }
+                 } else {
+                     this.inWaterTime = -1;
+                 }
+diff --git a/src/main/java/net/minecraft/world/entity/projectile/ProjectileUtil.java b/src/main/java/net/minecraft/world/entity/projectile/ProjectileUtil.java
+index 60b1e42f7c6c358ed316e014d026005a1ec5bd0a..9728702cf081b9eec329a3a692fdfcdf120e7606 100644
+--- a/src/main/java/net/minecraft/world/entity/projectile/ProjectileUtil.java
++++ b/src/main/java/net/minecraft/world/entity/projectile/ProjectileUtil.java
+@@ -128,6 +128,7 @@ public final class ProjectileUtil {
+     }
+ 
+     public static InteractionHand getWeaponHoldingHand(LivingEntity entity, Item item) {
++        if (Items.BOW == item) return InteractionHand.MAIN_HAND; // MONUMENTA - Only allow mobs to use bows from mainhand
```

Comment:
If this fails to match the main hand, it still runs the line below and returns the off hand. Maybe always return main hand instead?